MRI2CBCT/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+#-----------------------------------------------------------------------------
+set(MODULE_NAME MRI2CBCT)
+
+#-----------------------------------------------------------------------------
+set(MODULE_PYTHON_SCRIPTS
+  ${MODULE_NAME}.py
```

Comment:
You must add the utils scripts here. Currently CMake doesn't package those files so the scripts will not be available when published on the extension index.

```
utils/Method.py
utils/Preprocess_CBCT_MRI.py
utils/Preprocess_CBCT.py
utils/Preprocess_MRI.py
utils/Reg_MRI2CBCT.py
utils/utils_CBCT.py
```

Reply:
I am unable to test locally because of these files missing - I've added the files and will try again.

I don't currently have a build of Slicer Preview on my machine to support SlicerConda, so I need to rebuild that first. It will take a while. I'll follow up here if there are any other issues that prevent installation/packaging.

Reply:
I've made the change in the cmake list. 

I forgot we have to build it the extension to test it sorry. I'm gonna check if I have a build of Slicer Preview but I don't think so